packages/account/src/Components/forms/scroll-to-field-with-error.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { useFormikContext } from 'formik';
+
+type TScrollToFieldWithError = {
+    fields_to_scroll_top?: string[];
+    fields_to_scroll_end?: string[];
+    should_recollect_inputs_names?: boolean;
+};
+
+/**
+ * A ScrollToFieldWithError is a helper component for scrolling to field with error.
+ *  This component is supposed to be used with Formik and working validation as it uses formik errors object.
+ */
+
+const ScrollToFieldWithError = ({
+    fields_to_scroll_top,
+    fields_to_scroll_end,
```

Comment:
What are these props for like what would be usecase for top or end?

Reply:
in some of our pages scroll to center works not as expected. so we can adjust scrolling for exact fields on exact pages.